### ISSUE TITLE
Port forcing dates as timestamps to 1.3.x

### ISSUE THREAD

**Issue by leason805** [spring-cloud/spring-cloud-bus#200]
I had included spring-cloud-bus:1.3.3-RELEASE in my project. But i found that the behavior of ObjectMapper was changed by comparing without using spring cloud bus.

I have a very simple case like below

```
public class TestDomain {
        private String name;
        private Date createTime;
}


@RestController
public class TestController {
	
	@RequestMapping("/test")
	public TestDomain test(){
		TestDomain domain = new TestDomain();
		domain.setName("Test");
		domain.setCreateTime(new Date());
		return domain;
	}
}
```

 
When accessing http://localhost:8080/test, what i expect it should be
```
{"name":"Test","createTime":1560956260744}
```
but i found after included spring cloud bus, the result looks like
```
{"name":"Test","createTime":"2019-06-19T15:00:35.124+0000"}
```
and i traced the code, found that BusJacksonAutoConfiguration has created a bean BusJacksonMessageConverter, which changed some behavior of the global injected ObjectMapper

```
	public void afterPropertiesSet() throws Exception {
		this.mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
		this.mapper.registerModule(new SubtypeModule(findSubTypes()));
	}

```
Actually, what i want is keep the ObjectMapper work as usual, how can i do that? 
I tried to create a higher precedence AutoConfiguration by  extending BusJacksonAutoConfiguration and override busJsonConverter() method, to create BusJacksonMessageConverter by passing a null parameter, but failed since the BusJacksonMessageConverter is unvisiable, so what's the suggested way to do that?

BTW,what's the purpose to set SerializationFeature.WRITE_DATES_AS_TIMESTAMPS as false?

**Comment by spencergibb** [comment]
Please learn how to properly [format code and logs](https://help.github.com/articles/creating-and-highlighting-code-blocks/).

See #147

**Comment by leason805** [comment]
i saw from 2.x something has changed, but still hard code the timestamp serialization setting, so what's the suggest way to avoid this if i upgrade to 2.x?

**Comment by leason805** [comment]
thanks

**Comment by spencergibb** [comment]
I marked this as a bug to fix on the 1.3.x branch, reopening.

**Comment by spencergibb** [comment]
Closing as 1.x is no longer maintained
